tree/treeviewer/inc/TParallelCoordVar.h

```diff
@@ -12,8 +12,8 @@
 #ifndef ROOT_TParallelCoordVar
 #define ROOT_TParallelCoordVar
 
-#include "TAttLine.h"
 #include "TNamed.h"
+#include "TAttLine.h"
```

Comment:
Why this change. Shouldn’t we order alphabetically?

Reply:
I always sort includes in logical order - first base class(es), then all others.
And base classes listed in order they appears in the code.

Reply:
Doesn’t that presume deep knowledge of the header content? The preprocessor will expand them for you when visiting the transitive includes.

Usually the common/unwritten rule is by alphabetical order to avoid adding the same header twice and ordered by generality to avoid making headers non-standalone.

Reply:
In 99% of usecases there are few includes in header files. 
In such case alphabetical order does not matter.